examples/eliza/tutorial_eliza.ipynb

```diff
@@ -0,0 +1,120 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Eliza Example\n",
+    "\n",
+    "This example will walkthrough the steps to set up a Eliza chatbot which generates responses based on Eliza rules. We will utilize `stave` to visualize the dialogue page that allows you to chat with the bot interactively.\n",
+    "\n",
+    "###  Introduction\n",
+    "\n",
+    "Eliza chatbot is a famous rule-based chatbot invented in 1964. The rule-based and model-less nature makes it extremely suitable for demonstration purposes. This tutorial is based on `ElizaProcessor`. For more details, refer to https://github.com/asyml/forte/blob/master/forte/processors/nlp/eliza_processor.py.\n",
+    "\n",
+    "### Install Dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "pip install git+https://github.com/asyml/forte.git@master\n",
+    "pip install stave==0.0.2.dev1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Start a Eliza pipeline service \n",
+    "\n",
+    "Run the following python script to start a pipeline service to process input queries based on Eliza rules."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from forte.pipeline import Pipeline\n",
+    "from forte.data.data_pack import DataPack\n",
+    "from forte.data.readers import RawDataDeserializeReader\n",
+    "from forte.processors.nlp import ElizaProcessor\n",
+    "\n",
+    "\n",
+    "def start_eliza_service(\n",
+    "    input_format: str = \"DataPack\", service_name: str = \"test_name\"\n",
+    ") -> None:\n",
+    "    \"\"\"\n",
+    "    Start a remote service for ElizaProcessor\n",
+    "    \"\"\"\n",
+    "    pipeline = Pipeline[DataPack]()\n",
+    "    pipeline.set_reader(RawDataDeserializeReader())\n",
+    "    pipeline.add(ElizaProcessor())\n",
+    "    pipeline.serve(input_format=input_format, service_name=service_name)\n",
```

Comment:
We need to explain what this piece of code is doing. For example, this setup a pipeline served at port 8008, we should explain these.